UI/win-dll-blocklist.c

```diff
@@ -124,8 +124,8 @@ static blocked_module_t blocked_modules[] = {
 	// Wacom / Other tablet driver, locks up UI
 	{L"\\wintab32.dll", 0, 0, TS_IGNORE},
 
-	// Adobe Dynamic Link (Adobe CC), crashes in its own thread
-	{L"\\mc_trans_video_imagescaler.dll", 0, 0, TS_IGNORE},
+	// MainConcept Image Scaler, crashes in its own thread
+	{L"\\mc_trans_video_imagescaler.dll", 0, 1455495131, TS_LESS_THAN},
```

Comment:
Seems like it might be appropriate to #define the version number or at least add a comment as to how the number is calculated

Reply:
It's a unix timestamp, and the next perameter (`TS_LESS_THAN`) determines the matching rule.

In this case, it's set to `UTC Monday, February 15, 2016` (ie. DLL older than Feb 2016 gets blocked)

Reply:
This is documented in the struct:

https://github.com/obsproject/obs-studio/blob/63e4cdafad3d44a8a605c055b68ae2665401720c/UI/win-dll-blocklist.c#L71-L75

Reply:
Ah I suppose that's fine then

Reply:
I added a comment anyway just for consistency with all the others that have a timestamp.